This will be fixed in 2.6.28-rc1.


ERROR: "__bad_udelay" [drivers/net/r6040.ko] undefined!
make[1]: *** [__modpost] Error 1

diff --git a/drivers/net/Kconfig b/drivers/net/Kconfig
index e706744..ce85d44 100644
--- a/drivers/net/Kconfig
+++ b/drivers/net/Kconfig
@@ -1613,7 +1613,7 @@ config 8139_OLD_RX_RESET
 
 config R6040
 	tristate "RDC R6040 Fast Ethernet Adapter support (EXPERIMENTAL)"
-	depends on NET_PCI && PCI
+	depends on NET_PCI && PCI && (!ARM || BROKEN)
 	select CRC32
 	select MII
 	help
